activerecord/lib/active_record/railtie.rb

```diff
@@ -31,6 +32,8 @@ class Railtie < Rails::Railtie # :nodoc:
     config.active_record.sqlite3 = ActiveSupport::OrderedOptions.new
     config.active_record.sqlite3.represent_boolean_as_integer = nil
 
+    config.active_record.queues = ActiveSupport::InheritableOptions.new(destroy: :active_record_destroy)
```

Comment:
I guess it depends on what we want to expose as the public API. E.g. just the `ActiveRecord::DestroyJob` and the others to inherit from, or a succinct setting for many queues.

For the latter, something like this is now possible:

```ruby
# config/initializers/queues.rb
Rails.application.config.active_record.queues[:destroy] = …
Rails.application.config.active_storage.queues[:purge] = …
```

Reply:
We will have to end up exposing both because even that we ship with a job to destroy record later I already see a problem where your application requires a more complex job than the one we ship. Shopify for sure will need a custom job because of our infra.